release-notes/release-notes53.md

```diff
@@ -29,6 +29,45 @@ The following new features and enhancements were added for the {{site.productnam
 
 The following premium plugins updates were released alongside {{site.productname}} 5.3.
 
+### Enhanced Media Embed
+
+The {{site.productname}} 5.3 release includes the following accompanying changes affecting **Self-hosted Essential, Professional, Premium, and Enterprise customers only**:
+
+- The Java server-side component (`ephox-hyperlinking.war`) has been updated to version 2.3.0. This version requires Java 8 or higher. For information on the deprecation of Java 7 support, see: [Deprecated features](#deprecatedfeatures).
+- From {{site.productname}} 5.3, premium self-hosted bundles include the files required to deploy the server-side components as Docker containers. For information on Docker deployments of the server-side components, see: [Containerized service deployments]({{site.baseurl}}/enterprise/server/dockerservices/).
```

Comment:
"build and deploy" maybe? as they need to build it first, before they can deploy.

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/1482/commits/a371f8e8bf84dbb8784ef55e828d1af6ddb5b69f